style.css

```diff
@@ -0,0 +1,90 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@100&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", Times, serif;
+}
+body {
+  min-height: 100vh;
+
+  display: flex;
+  justify-content: center;
+  align-self: center;
+
+  background: #222;
+}
+.container {
+  margin: 40px;
+
+  position: relative;
+  max-width: 100%;
+
+  display: grid;
+  grid-template-columns: repeat(4, minmax(300px, 1fr));
+  grid-template-rows: minmax(100px, auto);
+  grid-auto-flow: dense;
+  grid-gap: 10px;
+}
+.container .box {
+  padding: 20px;
+
+  display: grid;
+  place-items: center;
+
+  background: #333;
+  color: #fff;
+
+  font-size: 20px;
+  font-weight: bolder;
+  text-align: center;
+  transition: 0.5s;
+}
+.container .box:hover {
+  background: #e91e63;
+}
+.container .box img {
+  position: relative;
+  max-width: 100px;
+  margin: 0 0 10px 0;
+}
+.container .box:nth-child(1) {
+  grid-column: span 2;
+  grid-row: span 1;
+}
+.container .box:nth-child(2) {
+  grid-column: span 1;
+  grid-row: span 2;
+}
+.container .box:nth-child(3) {
+  grid-column: span 1;
+  grid-row: span 1;
+}
+.container .box:nth-child(4) {
+  grid-column: 4/5;
+  grid-row: span 1;
+}
+.container .box:nth-child(5) {
+  grid-column: 2/5;
+  grid-row: span 2;
+}
+.container .box:nth-child(6) {
+  grid-column: 2/3;
+  grid-row: 2/3;
+}
+.container .box:nth-child(7) {
+  grid-column: span 1;
+  grid-row: span 3;
+}
+
+@media (max-width: 990px) {
+  .container {
+    grid-template-columns: repeat(auto-fill, minmax(50%, 1fr));
+    grid-template-rows: minmax(auto, auto);
+  }
+  .container .box {
+    grid-row: unset !important;
+    grid-column: unset !important;
```

Comment:
Why do we need `!important` declaration here ??

Reply:
because wanted to make the ```unset``` value applied no matter what